spec/requests/schools_request_spec.rb

```diff
@@ -48,4 +48,14 @@
     expect(response).to have_http_status(:success)
   end
 
+  it 'returns a successful response for school by partial name' do
```

Comment:
this should be added to https://github.com/openstax/openstax-salesforce-api/blob/master/spec/integration/v1/schools_spec.rb 

Here is an example of searching in an rswag block.
https://github.com/rswag/rswag/blob/0def9fad6fbfc97168d0e1ffc740c115e00c0768/test-app/spec/integration/blogs_spec.rb#L31-L49

Reply:
All tests moved to Swagger test

Reply:
Done